CONTRIBUTING.md

```diff
@@ -12,6 +14,23 @@ Run tests:
 npm test
 ```
 
+## Code
+
+This repository is a monorepo which means that it contains multiple npm packages.
+
+Running `npm install` in the repository's root directory will install the dependencies of all packages and link the packages that depend on each other.
+
+Running `npm test` in the repository's root directory will run all packages' tests. Run `npm test` in a package to test this package only.
+
+Just edit the code like you always do. Be aware that we automatically lint the code according to [JavaScript standard code style](https://github.com/feross/standard), so stick to the code style or expect the linter to shout at you.
+
+When adding a new package you might need to run `npm run postinstall` in the repository's root directory to update the links between packages depending on each other.
+
+
+## Share
+
+Commit your changes, push them and open a pull request. We will review your pull request and eventually merge your valuable contribution back into the mother repository.
```

Comment:
Sure, but I wanted to keep it short ^^

Btw, dropped the "unfriendly" phrase 😉